### ISSUE TITLE
Automatic selection of dropped entities makes link hierarchy confusing

### ISSUE THREAD

**Issue by tobias-ourmachinery** [OurMachinery/themachinery-public#282]
When you drag an entity from the asset browser into the scene tab the dropped entity is parented under the currently selected entity in the tree view, which probably makes sense. But since the newly dropped entity automatically gets selected repeating the operation makes the link hierarchy kind of confusing as each new entity dropped gets parented under the last entity dropped. 

Suggestion: Stop automatically select the newly dropped entity. Keep the old selection on drop.

**Comment by karl-ourmachinery** [comment]
I have changed this for now so that the root object is always used as parent. Initially I tried just not selecting the new object, but that turned out annoying because you often want to change properties of the newly added object.

We may want to add some sort of "drag n drop target", so that any entity can be used as drop target, while still changing the selection to the newly added entity. Leaving this open.
